app/javascript/components/request-workflow-status/data.js

```diff
@@ -43,13 +43,43 @@ const convertDate = (date) => {
   return formattedDate.toString();
 };
 
+// Converts the duration time in ms and returns a string in format: w days x hours y minutes z seconds
+// duration: time in ms
+const convertDuration = (duration) => {
+  const durationString = moment.duration(duration, 'milliseconds').toISOString().split('PT')[1];
+  let startIndex = 0;
+  let resultString = '';
+  if (durationString.indexOf('H') >= 0) {
+    resultString += `${durationString.slice(startIndex, durationString.indexOf('H') + 1)} `;
+    startIndex = durationString.indexOf('H') + 1;
+  }
+  if (durationString.indexOf('M') >= 0) {
+    resultString += `${durationString.slice(startIndex, durationString.indexOf('M') + 1)} `;
+    startIndex = durationString.indexOf('M') + 1;
+  }
+  if (durationString.indexOf('S') >= 0) {
+    resultString += `${durationString.slice(startIndex, durationString.indexOf('S') + 1)} `;
+    startIndex = durationString.indexOf('S') + 1;
+  }
+  return resultString;
+};
+
+const getItemIcon = (item) => {
+  if (item.RunnerContext && item.RunnerContext.success) {
+    return { icon: 'carbon--CheckmarkOutline', props: { style: { color: '#2d7623' } } };
+  } if (item.RunnerContext && item.RunnerContext.Error) {
+    return { icon: 'carbon--MisuseOutline', props: { style: { color: '#cc0000' } } };
+  }
+  return { icon: 'carbon--PlayOutline', props: { style: { color: '#ec7a08' } } };
```

Comment:
I think it might make more sense to use CSS here instead of inline styles?  I don't like the hardcoding of the colors if we have to change them later, it would be better in CSS.

Reply:
Yeah I can change this, I saw the miq-data-table component takes in colour as a style prop so I thought we should use that but if css is preferred I can make the change